Library/Homebrew/cmd/list.rb

```diff
@@ -116,8 +116,14 @@ def list
       ls_args << "-r" if args.r?
       ls_args << "-t" if args.t?
 
-      safe_system "ls", *ls_args, HOMEBREW_CELLAR unless args.cask?
-      list_casks(args: args) unless args.formula?
+      unless args.cask?
+        ohai "Formulae:" if $stdout.tty?
+        safe_system "ls", *ls_args, HOMEBREW_CELLAR
```

Comment:
I wonder if we should deprecate these `ls_args` if we're not going to do them for casks?

Reply:
I find these `ls_args` kinda useful, and I'm not sure they have a clear analogue for casks. Though I suppose you can get them and more by using `ls` in `HOMEBREW_CELLAR`.

Reply:
We could be doing `ls /usr/local/Caskroom/`

Reply:
Are those flags as useful when applied to `/usr/local/Caskroom` as they are for `HOMEBREW_CELLAR`? Genuine question; I actually don't know. I'll look into it.

Reply:
They look equally useful applied to either to me. I'd argue using a consistent method of outputting both would be good (i.e. `ls` for both or neither)

Reply:
Ok, I think I'm happy to use `ls` for both then. I'll do that in a follow-up PR so we can get adequate feedback from Cask maintainers.